pkg/plugin/v3/scaffolds/internal/templates/config/rbac/role_binding.go

```diff
@@ -22,15 +22,15 @@ import (
 	"sigs.k8s.io/kubebuilder/pkg/model/file"
 )
 
-var _ file.Template = &ManagerRoleBinding{}
+var _ file.Template = &RoleBinding{}
 
-// ManagerRoleBinding scaffolds the config/rbac/role_binding.yaml file
-type ManagerRoleBinding struct {
+// RoleBinding scaffolds a file that defines the role binding for the manager
+type RoleBinding struct {
```

Comment:
The purpose of this scaffold is clearer as `ManagerRoleBinding` imo

Reply:
have we any other role_binding or just this one used by the manager? Since I do not think that we have another then, I would be ok with the change in the name. It is up to you folks.

Reply:
This is the main role binding. We have one for the auth proxy wich is called `AuthProxyRoleBinding`. Again the changed name reflects the file name, IIRC.